### ISSUE TITLE
Not waking from sleep anymore

### ISSUE THREAD

**Issue by CV8R** [git-developer/autosuspend#11]
Hi,

Was working fine for sometime now I am missing recordings because the system isn't waking for scheduled recordings.  Something has changed in the system now, not sure if its something on the machine or in tvheadend.

The syslog reports:

`autosuspend[7456]: Fetched next 0 upcoming recordings from Tvheadend`

I followed this issue instructions: [https://github.com/git-developer/autosuspend/issues/1](url)

`sudo/home/tvheadend/Downloads/test-autosuspend-function.sh IsTvheadendBusy` 

returns the attached into the terminal

[output.txt](https://github.com/git-developer/autosuspend/files/2038053/output.txt)

When issuing the curl statement

`sudo curl -s --user "username:password" "http://127.0.0.1:9981/api/dvr/entry/grid_upcoming?sort=start_real&dir=ASC&limit=5" >response.js`

response.json contains attached:

[response.json.txt](https://github.com/git-developer/autosuspend/files/2038055/response.json.txt)

Thanks in advance for the help!

CV8R

**Comment by git-developer** [comment]
Your output files do not contain a valid TVHeadend response file but a HTML page. Please check if TVheadend is up and running on `http://127.0.0.1:9981/`. Are you able to access the TVHeadend web frontend with a browser? Maybe you run a web server that doesn't redirect to TVHeadend.

**Comment by CV8R** [comment]
Arrrghhh - as soon as I read your post I knew what the cause was.  I have changed the root URL because I use a reverse proxy to access the server from the WAN side.  I did this a few weeks ago and simply forgot that I changed tvh.  That means that the server is now accessed via http://127.0.0.1:9981/tvh. 

Sorry, I am a little annoyed with myself that I didn't pick that up - was testing from the WAN side using https and SSH so didn't realise. My bad.

Anyway, it's highlighted another issue.

I changed the webroot per these instructions by modifying /etc/default/tvheadend

[https://tvheadend.org/boards/4/topics/10626](url)

When i change the curl to:

`sudo curl -s --user "username:password" "http://127.0.0.1:9981/tvh/api/dvr/entry/grid_upcoming?sort=start_real&dir=ASC&limit=5" >response.js`

Data is returned as expected.

I took a brief look at the config file for autosuspend and it looks like it only has config for IP Address and Port instead of a URL and assumed it's probably appending the two.

I appended /tvh to the port value 

`TVHEADEND_HTTP_PORT=9981/tvh`

On quick tests using the test-autosuspend-function.sh it is returning correct values.  I have set autosuspend to sleep the machine again and will see how it goes and shall report back.

Ideally there would be a config item for changing the URL.

Cheers
CV8R
